includes/admin/abstract/class-rop-services-abstract.php

```diff
@@ -904,4 +904,54 @@ function( $d ) {
 		return $valid;
 	}
 
+	/**
+	 * Capture logs on ROP API.
+	 *
+	 * @since   9.1.3
+	 * @access  public
+	 *
+	 * @return bool
+	 */
+	public function save_logs_on_rop( $data = array() ) {
```

Comment:
A small suggestion: since we are not using the return value in any flow, you can remove the `return` of bool and mark it as void in the PHPDocs function definition.